draw/src/main/java/chat/rocket/android/draw/main/ui/DrawActivity.kt

```diff
@@ -1,128 +1,167 @@
-package chat.rocket.android.draw
+package chat.rocket.android.draw.main.ui
```

Comment:
why `.main`?

Reply:
I was not too much creative to define that package name. I just left `.main` because it is the "place"  of the main classes (activity) in that module.